File: Praktikum I Modul 5/Modul 5/Python/PRAK503-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
# fungsi
def maksimal(maxs, num):
    if (maxs < num):
        maxs = num
            
    return maxs
# fungsi
def minimal(minims, num):
    if (minims > num):
        minims = num

    return minims

# deklarasi variabel
batas = 0
maks = -100000
minim = 100000
# input bilangan
bilangan = int(input())

# perulangan
while(batas < bilangan):

    # input nilai
    nilai = int(input())

    maks = maksimal(maks, nilai)
    minim = minimal(minim, nilai)

    # increment
    batas += 1

# output
print(f"{maks} {minim}")

File: Praktikum I Modul 5/Modul 5/C/PRAK503-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>

// fungsi
int maksimal(int max, int num){
    // kondisi
    if (max < num){
        max = num;

        return max;
    } 
}

// fungsi
int minimal(int min,int num){
    // kondisi
    if (min > num){
        min = num;

        return min;
    }
}

int main(){
    // deklarasi variabel
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan, nilai;

    // input bilangan
    scanf("%d", &bilangan);

    // perulangan
    while (batas < bilangan){
        // input nilai
        scanf("%d", &nilai);
        
        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);\

        // increment variabel batas
        batas++;
    }

    // output
    printf("%d %d", maks, minim);

    return 0;
}